scripts/cnode-helper-scripts/guild-deploy.sh

```diff
@@ -238,7 +238,7 @@ os_dependencies() {
       fi
     elif [[ "${DISTRO}" =~ Fedora ]]; then
       #Fedora
-      pkg_opts="${pkg_opts} --allowerasing"
+      if ${pkgmgrcmd} -h  | grep -q "\--allowerasing"; then pkg_opts="${pkg_opts} --allowerasing"; fi
```

Comment:
Same as above comment for dnf v4 vs. v5:

```suggestion
      if ${pkgmgrcmd} install -h  | grep -q "\--allowerasing"; then pkg_opts="${pkg_opts} --allowerasing"; fi
```